core/src/test/java/io/grpc/internal/ServerImplTest.java

```diff
@@ -420,7 +420,6 @@ public void start(ServerListener listener) throws IOException {
     } catch (IOException e) {
       assertSame(ex, e);
     }
-    verifyNoMoreInteractions(executorPool);
```

Comment:
Why was this removed?

Reply:
Because `executorPool.getObject()` is called after the change.